packages/blade/src/components/TopNav/_decisions/decisions.md

```diff
@@ -0,0 +1,291 @@
+**Ans:** The product navigation items will be moved to the bottom of the screen, but this will be part of a separate component called `BottomNavigation` which we will ship in phase 2.
+
+Mobile Navbar example:
+
+```jsx
+const Dashboard = () => {
+  const { isMobile } = useBreakpoint();
+
+  if (isMobile) {
+    return (
+      <TopNav>
+        <TopNavBrand />
+        <TopNavContent>
+          <Button icon={MenuIcon} variant="tertiary" accessibilityLabel="Toggle side navigation" />
+          <Text>{currentPageTitle}</Text>
+        </TopNavContent>
+        <TopNavActions>
+          <Button icon={ActivityIcon} accessibilityLabel="View Ecosystem Health" />
+          <Button icon={AnnouncementIcon} accessibilityLabel="View Announcements" />
+          <Button icon={RayIcon} accessibilityLabel="Ask Ray" />
+          <Avatar onClick={openBottomSheet} />
+          <BottomSheet>
+            { /* Avatar BottomSheet Content */ }
+          </BottomSheet>
+        </TopNavActions>
+      </TopNav>
+    )
+  }
+
+  return (
+    // ... desktop navigation bar jsx
+  )
+}
+```
+
+## Accessibility
+
+- TabNav
+  - Tab items will have `role=link` instead of `tab` because they don't contain any related tab panels.
+  - Tab nav items will be a set of links that will be navigable using the keyboard.
+  - The selected tab will have an `aria-selected` attribute set to `true`.
+
+- TopNav
+  - The navigation bar will have `role=navigation` to indicate that it is a navigation landmark.
+  - TopNav sub components will only compose other components and will not have any ARIA roles or attributes.
+
+## References
+
+- [NavBar - NextUI](https://nextui.org/docs/components/navbar)
+- [AppBar - MUI](https://mui.com/material-ui/react-app-bar/)
+- [FullscreenNav - Shopify](https://polaris.shopify.com/components/navigation/fullscreen-bar)
+- [Navigation - Atlassian DS](https://atlassian.design/components/atlassian-navigation/examples)
+- [PageHeader - Primer](https://primer.style/components/page-header/react/draft)
+- [TabNav - Primer](https://primer.style/components/tab-nav/react/alpha)
+- [HeaderNav - Carbon DS](https://react.carbondesignsystem.com/?path=/story/components-ui-shell-header--header-w-navigation-and-actions)
+- [HeaderNav - BaseWeb DS / Uber](https://baseweb.design/components/header-navigation/#basic-link1)
+- [TopNav - Spectrum / Adobe](https://opensource.adobe.com/spectrum-web-components/components/top-nav/)
+
+## Open Questions
+
+### Dev
+
+1. How will the Menu component integrate with the BottomSheet?
+2. How will the TabNav component handle the chevron icon for the Menu component?
+3. Should we call this component `TopNav`/`TopNavigationBar` instead of `NavigationBar`?
+   1. We will call it `TopNav` since we will also have a `BottomNav` in the future.
+4. For the responsive behavior, should we just hide the `TopNavBrand` and `TopNavContent` with CSS media queries? instead of asking consumers to do conditional rendering?
```

Comment:
We should handle the responsiveness ourselves.

Reply:
That's a bit tricker to do, because of the nature of the TopNav. 

The TopNav is quite flexible and mostly provides a skeleton of UI layout where consumers can add logo, content, footer actions etc. (if you remove the TabNav)

For example, the consumer is in charge of rendering the SearchInput in the footer so we cannot remove it from our end, consumer will use the responsive styled props to remove it on mobile devices. 

Tho few things that we do: 

1. We hide TopNavBrand internally on mobile because there is not enough space at top to show logo/actions + content all at once
2. We adjust padding/margins accordingly to ensure in mobile all the footer actions/content align properly. 
3. Expose responsive props in TopNav components so consumers can dynamically add responsive styles.